ansible/playbooks/roles/kubernetes_master/tasks/main.yml

```diff
@@ -28,63 +24,73 @@
         - import_tasks: copy-kubernetes-pki.yml
         - import_tasks: master-join.yml
 
-- name: Collect current apiserver certificate 'not_after' date by openssl
-  command: openssl x509 -enddate -noout -in apiserver.crt
+- name: Collect subject alternative names of apiserver certificate
+  shell: |-
+    set -o pipefail && \
+    openssl x509 -in /etc/kubernetes/pki/apiserver.crt -text -noout \
```

Comment:
How about using `community.crypto.x509_certificate_info` module like we have in `opendisto_for_elasticsearch` role?

Reply:
Decided to leave as it is for now, there is #1605 task for refactoring.